news/views.py

```
@@ -669,6 +685,231 @@ def post(self, request, *args, **kwargs):
+        post_type = request.POST.get("post_type", "")
+        block_config = self._POST_BLOCK_MAP.get(post_type, None)
+        form_class = self._POST_TYPE_MAP.get(post_type)
+
+        if block_config is None or form_class is None:
+            return self.error_message_and_render(
+                message=_(
+                    "Invalid post type selected. Please choose a valid post type."
+                )
+            )
+
+        block_name, block_class = block_config
```

Ruff (0.16.0)

[warning] Unpacked variable `block_class` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)